### PR TITLE
Make header suite run on d.ts files instead of tsconfig

### PR DESCRIPTION
**Pull request by rakatyal**

### CODE REVIEW

File: src/util/util.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import * as fs from 'fs';
 import * as Lazy from 'lazy.js';
 import * as Promise from 'bluebird';
 import * as globMod from 'glob';
+var grace_fs = require('graceful-fs');
```
Comment (RyanCavanaugh) [inline src/util/util.ts]: What's the change here for?

Comment (rakatyal) [inline src/util/util.ts]: 'fs' gave me EMFILE error saying too many files open when I ran it on all the .d.ts files. This uses incremental back off when such an error is encountered.

Comment (RyanCavanaugh) [inline src/util/util.ts]: :+1:

**Comment by RyanCavanaugh** [comment]
:+1:

**Pull request merged**